schemas/common/s3/partial/v3.schema.json

```
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://mapcolonies.com/common/s3/partial/v3",
+  "type": "object",
+  "required": ["accessKeyId", "secretAccessKey", "host"],
+  "properties": {
+    "accessKeyId": {
+      "type": "string",
+      "description": "The access key id of the bucket",
+      "x-env-value": "S3_ACCESS_KEY"
+    },
+    "secretAccessKey": {
+      "type": "string",
+      "description": "The secret access key of the bucket",
+      "x-env-value": "S3_SECRET_KEY"
+    },
+    "host": {
```

it should be called "endpoint" instead "host"

```suggestion
    "endpoint": {
```

---

Changed